plugins/modules/hmc_user.py

```diff
@@ -737,7 +743,15 @@ def ensure_update(module, params):
                     filter_d['NAMES'] = attributes['new_name']
                 changed = True
 
+        if m_config["NAME"] == 'hscroot' and m_config["passwd"] is not None and hmc_user == 'hscroot':
+            password = m_config["passwd"]
+            hmc_conn = HmcCliConnection(module, hmc_host, hmc_user, password)
```

Comment:
Please add a comment why we are re-initializing the HmcCliConnection?

Reply:
Done